azurerm/internal/services/loganalytics/tests/log_analytics_data_export_resource_test.go

```diff
@@ -0,0 +1,272 @@
+		CheckDestroy: testCheckAzureRMLogAnalyticsDataExportRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMLogAnalyticsDataExportRule_complete(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMLogAnalyticsDataExportRuleExists(data.ResourceName),
+				),
+				ExpectNonEmptyPlan: true, // Due to API changing case of attributes you need to ignore a non-empty plan for this resource
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func testCheckAzureRMLogAnalyticsDataExportRuleDestroy(s *terraform.State) error {
+	conn := acceptance.AzureProvider.Meta().(*clients.Client).LogAnalytics.DataExportClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_log_analytics_data_export_rule" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		workspace, err := parse.LogAnalyticsWorkspaceID(rs.Primary.Attributes["workspace_resource_id"])
+		if err != nil {
+			return nil
+		}
+
+		resp, err := conn.Get(ctx, resourceGroup, workspace.Name, name)
+		if err != nil {
+			return nil
+		}
+
+		if resp.StatusCode != http.StatusNotFound {
+			return fmt.Errorf("Log Analytics Data Export Rule still exists:\n%#v", resp)
+		}
+	}
+
+	return nil
+}
+
+func testCheckAzureRMLogAnalyticsDataExportRuleExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := acceptance.AzureProvider.Meta().(*clients.Client).LogAnalytics.DataExportClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		// Ensure we have enough information in state to look up in API
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Not found: %s", resourceName)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup, hasResourceGroup := rs.Primary.Attributes["resource_group_name"]
+		if !hasResourceGroup {
+			return fmt.Errorf("Bad: no resource group found in state for Log Analytics Data Export Rule: %q", name)
+		}
+
+		workspace, err := parse.LogAnalyticsWorkspaceID(rs.Primary.Attributes["workspace_resource_id"])
+		if err != nil {
+			return fmt.Errorf("Bad: unable to access 'workspace_resource_id' for Log Analytics Data Export Rule: %q", name)
+		}
+
+		resp, err := conn.Get(ctx, resourceGroup, workspace.Name, name)
+		if err != nil {
+			return fmt.Errorf("Bad: Get on Log Analytics Data Export Rule Client: %+v", err)
+		}
+
+		if resp.StatusCode == http.StatusNotFound {
+			return fmt.Errorf("Bad: Log Analytics Data Export Rule %q (resource group: %q) does not exist", name, resourceGroup)
+		}
+
+		return nil
+	}
+}
+
+func testAccAzureRMLogAnalyticsDataExportRule_template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-lader-%d"
```

Comment:
we usually just put the service here
```suggestion
  name     = "acctestRG-la-%d"
```

but this is ok tbh